.jenkins/pytorch/build.sh

```diff
@@ -210,6 +210,12 @@ if [[ "${BUILD_ENVIRONMENT}" == pytorch-linux-xenial-py3.6-gcc7-build || "${BUIL
   export USE_GLOO_WITH_OPENSSL=ON
 fi
 
+if [[ "${BUILD_ENVIRONMENT}" == pytorch-linux-xenial-py3* ]]; then
+  if [[ "${BUILD_ENVIRONMENT}" != *android* && "${BUILD_ENVIRONMENT}" != *ios* && "${BUILD_ENVIRONMENT}" != *mobile* && "${BUILD_ENVIRONMENT}" != *cuda* ]]; then
```

Comment:
Android and cuda should be sufficient, shouldn't it, as `ios` builds could not run on linux
```suggestion
  if [[ "${BUILD_ENVIRONMENT}" != *android* && "${BUILD_ENVIRONMENT}" != *cuda* ]]; then
```